app/services/LocationService.js

```diff
@@ -55,6 +58,17 @@ export class LocationData {
       let unixtimeUTC = Date.parse(nowUTC);
       let unixtimeUTC_28daysAgo = unixtimeUTC - 60 * 60 * 24 * 1000 * 28;
```

Comment:
Kindly use name of variables for deeper understood these numbers - `60 * 60 * 24 * 1000 * 28`